repository-data/application/src/main/resources/hcm-config/configuration/modules/scxmlregistry.yaml

```diff
@@ -0,0 +1,7 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:modules/scxmlregistry/hippo:moduleconfig/hipposcxml:definitions/documentworkflow:
+      hipposcxml:source:
```

Comment:
I will check tomorrow, we might need to add the ".meta.before" to ensure our workflow is before the default one.

Reply:
Ah ok, not sure about that, this is what auto-export spat out. Let me know what you find 🙂